sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -305,12 +307,10 @@ logger.setLevel(logging.DEBUG)
 handler = logging.StreamHandler(stream=sys.stdout)
 logger.addHandler(handler)
 
-# Configure a file output
-file_handler = logging.FileHandler(filename)
-logger.addHandler(file_handler)
+credential = DefaultAzureCredential()
 
 # Enable network trace logging. Each HTTP request will be logged at DEBUG level.
-client = KeyClient(vault_endpoint=url, credential=credential, logging_enable=True)
+client = KeyClient(vault_endpoint=<your-vault-url>, credential=credential, logging_enable=True)
```

Comment:
What do you think of using an obvious placeholder URL like "https://your-vault-url.vault.azure.net" (consistently, everywhere) so anyone reading this knows what their replacement should look like? I'm not saying it must change, I only want to collect other opinions as I consider my own.

My current thought is that showing how the value should look is worth the risk of someone trying to execute code with the placeholder, because in that case the code either hilariously succeeds (wow what are the odds) or produces a reasonable error.

Reply:
yeah i like that!